packages/realm/src/Configuration.ts

```diff
@@ -47,6 +50,15 @@ export type BaseConfiguration = {
   disableFormatUpgrade?: boolean;
   encryptionKey?: ArrayBuffer | ArrayBufferView | Int8Array;
   onMigration?: MigrationCallback;
+  /**
+   * A flag to indicate whether Realm should resolve
+   * embedded object constraints after migration. If this is `true` then all embedded objects
+   * without a parent will be deleted and every embedded object with multiple references to it
```

Comment:
Nit - is it "multiple"? Or would this apply even for a single reference (I think as the test implies)? May suggest saying: "...every embedded object with one or more references..."

Reply:
I plead guilty: I copied from Kotlin :smile:

Your formulation sounds more accurate, and I will update before merging.